qemu/tests/live_snapshot_transaction.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+from autotest.client.shared import error
+from qemu.tests import live_snapshot_basic
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    live_snapshot_transaction test:
+
+    1. Boot up guest with a system disk and 2 data disk.
+    2. Create multiple live snapshots simultaneously for all 3 disks with transaction.
+    3. Check guest which should boot up and reboot successfully.
+
+    :param test: Kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    arg_list = []
+    try:
```

Comment:
I think no need for this case, as there is exception handling.

Or where do you think it's a good place?